doc/database/hierarchy.md

```diff
@@ -92,14 +92,42 @@ Either two parent binomials, separated by a "x" or "×" or a given binomial, wit
 (see [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_name)).
 So the name does not necessarily say if a plant is a hybrid.
 
-### Rendering
+### Rendering the Unique Name
 
-The unique name must be rendered:
+The unique name of a plant needs to be displayed in the following manner:
 
-- Latin name in _italics_.
-- Cultivar name normal font in single quotes (as in database).
+- Latin name should be displayed in _italics_.
+- The cultivar name should be in regular font, enclosed in single quotes (as stored in the database).
```

Comment:
This isn't your fault, but since you are already updating this document could you maybe explain further above what the cultivar is?
That might be useful for new project contributors.

Reply:
There is an explanation of Cultivar in doc/architecture/12glossary.md